governance/xc_admin/packages/xc_admin_common/src/propose.ts

```diff
@@ -27,8 +27,11 @@ import { MultisigAccount } from "@sqds/mesh/lib/types";
 import { mapKey } from "./remote_executor";
 import { WORMHOLE_ADDRESS } from "./wormhole";
 import { TransactionBuilder } from "@pythnetwork/solana-utils";
+import { PACKET_DATA_SIZE_WITH_ROOM_FOR_COMPUTE_BUDGET } from "@pythnetwork/solana-utils";
+import { PriorityFeeConfig } from "@pythnetwork/solana-utils/lib/transaction";
 
-export const MAX_EXECUTOR_PAYLOAD_SIZE = PACKET_DATA_SIZE - 687; // Bigger payloads won't fit in one addInstruction call when adding to the proposal
+export const MAX_EXECUTOR_PAYLOAD_SIZE =
+  PACKET_DATA_SIZE_WITH_ROOM_FOR_COMPUTE_BUDGET - 687; // Bigger payloads won't fit in one addInstruction call when adding to the proposal
```

Comment:
The key change is we make more space by replacing PACKET_DATA_SIZE by PACKET_DATA_SIZE_WITH_ROOM_FOR_COMPUTE_BUDGET